src/adapter/src/coord/id_bundle.rs

```diff
@@ -41,6 +57,6 @@ impl CollectionIdBundle {
         self.storage_ids
             .iter()
             .copied()
-            .chain(self.compute_ids.iter().copied())
+            .chain(self.compute_ids.values().flat_map(BTreeSet::iter).copied())
```

Comment:
I'm a teensy uneasy about this because it could have been used with the assumption of a single compute instance, which is no longer true. Are bundles ever used to do specific work on a single instance? Can you either look for uses of this or like...change the signature to require an instance id, just to double check that all call sites are now safe (but not commit that change).

Reply:
Here are all the uses, I think they are all safe:
- In `determine_timestamp` used to validate that all IDs belong to the same timeline.
- In `determine_timestamp` used to see if any ID belongs to a table.
- In `sequence_peek` used (twice) to generate an error message if a query involves IDs outside of the time domain.
- In `sequence_explain_old` used to see if any ID belongs to a table.

Reply:
Great. Thanks for checking.

Reply:
np, thanks for the detailed review!